backend/apps/contact/models.py

```diff
@@ -1,12 +1,12 @@
 from django.db import models
-
+from phonenumber_field.modelfields import PhoneNumberField
```

Comment:
Did u add requirements.txt too ?

Reply:
i forget to add that just need to add 
phonenumberslite==8.13.28

Reply:
Can better good job!